crypto/stark-verifier-ethereum/test/merkle_test.ts

```diff
@@ -1,100 +1,117 @@
-import { deployContract, solidity } from "ethereum-waffle";
-import { waffle } from "@nomiclabs/buidler";
-import chai from "chai";
-import {tx_to_events} from './test_utils';
+/* tslint:disable:custom-no-magic-numbers only-arrow-functions ordered-imports */
```

Comment:
I understand the `no-magic-numbers`, but why no arrow functions or sorted imports?

Reply:
Did some experiments, I think these two lints (`only-arrow-functions` & `ordered-imports`) are correct and we should enable and fix them.